web-ui/src/pages/AccessCodePage.jsx

```diff
@@ -6,30 +6,54 @@ import { Button } from "@mui/material";
 import "./AccessCodePage.css";
 
 function AccessCodePage() {
+
+  const nextPageHandler = () =>{
+
+  }
+
   return (
     <div>
+{/* <<<<<<< HEAD
+      <div className="page_body">
+        <div className="page_content">
+          <center>
+            <h1>Please Enter in your Access Code:</h1>
+          </center>
+          <Link to="/culturevideo">
+            <Button onClick = {nextPageHandler} variant="contained" size="large" id="cultureVideo">
+              Culture Video
+            </Button>
+          </Link>
+        </div>
+      </div>
+======= */}
       <center>
-        <h1>Please Enter in your Access Code:</h1>
-        <Box
-          component="form"
-          sx={{
-            "& > :not(style)": { m: 1, width: "50ch" },
-          }}
-          noValidate
-          autoComplete="off"
-        >
-          <TextField
-            id="outlined-basic"
-            label="Access Code:"
-            variant="outlined"
-          />
-        </Box>
+        <h1>Please Enter in your Access Code Below</h1>
       </center>
+
+      <div class="inputCode">Access Code:</div>
+
+      <div class="inputField">
+        <TextField
+          required
+          error
+          id="outlined-error-helper-text"
+          label="Error"
+          defaultValue="Input Code"
+          helperText="Incorrect entry."
+          size="small"
+        />
+      </div>
+
       <Link to="/culturevideo">
```

Comment:
Lets link this to the path /onboarding to get the button to link to the webportal when we click on it